nms/app/views/equipment/GatewayDetailSubscribers.tsx

```diff
@@ -51,15 +44,19 @@ export default function GatewayDetailSubscribers(props: GatewayDetailType) {
     refresh: props.refresh,
   });
   const subscriberCtx = useContext(SubscriberContext);
-  const gwSubscriberMap =
-    // $FlowIgnore
-    subscriberCtx.gwSubscriberMap[props.gwInfo?.device?.hardware_id] || [];
+  const hardware_id = props.gwInfo?.device?.hardware_id;
+  const gwSubscriberMap = hardware_id
+    ? subscriberCtx.gwSubscriberMap[hardware_id]
+    : [];
```

Comment:
I somehow wonder that this is OK: It looks like `hardware_id` should be a number or string, but than the alternative is an empty list?
But apparently it is ok. So never mind...

Reply:
I'm not sure if I can follow you :)